src/core/src/bootstrap/Constants.py

```diff
@@ -330,4 +333,5 @@ class UbuntuProClientSettings(EnumBackport):
         FEATURE_ENABLED = True
         MINIMUM_PYTHON_VERSION_REQUIRED = (3, 5)  # using tuple as we can compare this with sys.version_info. The comparison will happen in the same order. Major version checked first. Followed by Minor version.
         MAX_OS_MAJOR_VERSION_SUPPORTED = 18
+        MINIMUM_CLIENT_VERSION = "27.14.4"
```

Comment:
Is this refering to Pro Client? Can be more specific in naming?

Reply:
This is part of the  UbuntuProClientSettings already which implies the same. So, no specific naming required